app/Http/Controllers/SetupDocumentSignatureController.php

```
@@ -0,0 +1,54 @@
+        $signatures = SetupDocumentSignature::all()
+            ->groupBy('document_type');
+        return SetupDocumentSignatureResource::collection($signatures->flatten())
+            ->additional([
+            'success' => true,
+            'message' => 'Document signatures retrieved successfully.',
+            'grouped' => $signatures
+            ]);
+    }
+    public function store(StoreSetupDocumentSignatureRequest $request)
+    {
+        $signature = SetupDocumentSignature::create($request->validated());
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\SetupDocumentSignature' in method 'store'. (Clean Code Rules)

(StaticAccess)